unison-cli/src/Unison/Codebase/Editor/HandleInput/Merge2.hs

```diff
@@ -974,6 +974,8 @@ findConflictedAlias defns diff =
             g hashed1 alias =
               case Map.lookup alias diff of
                 Just (DiffOp'Update hashed2) | hashed1 == hashed2.new -> Nothing
+                -- If "foo" was updated but its alias "bar" was deleted, that's ok
+                Just (DiffOp'Delete _) -> Nothing
```

Comment:
Is there a symmetric case that needs to be considered; i.e. when "foo" was deleted but it's alias "bar" was updated?

Reply:
That symmetric case hits this code path to return `Nothing` (meaning "no problem") https://github.com/unisonweb/unison/pull/5067/files/528c2a1cd6930f36e957fc04543b1f3232f3febb..e3440a91c6832fb01a617eb48d7297df05af309f#diff-56bc494d4c38b12f3b10d9e6ac7aba28556377ede79ab99ccf723f1c91c17881R965